.travis.yml

```diff
@@ -1,5 +1,5 @@
 # File managed by pluginsync
-sudo: false
+sudo: true
```

Comment:
Switching to 'true' means you use a VM instead a container. I don't see any reason to use a VM here.